blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,606 @@
+
+<!--truncate-->
+
+## <TextGradient startColor="#2A6DFF" endColor="#245DD9">New Year - New Build</TextGradient>
+
+Now that the dust has settled down from [our biggest single release to date](../../2024/pcsx2-2-release/index.mdx), we decided to step back a bit and give more love and attention to the interface, Quality of Life features and Accessibility. There are also more performance improvements to be seen, which we are very excited to show you! Our contributors have been hard at work further refining the foundation that has been laid out since 2.0.0 to make PCSX2 more accessible to everyone!
+
+<!--PR List-->
+
+## <TextGradient startColor="#7828C8" endColor="#FF4ECD">Major Spotlight</TextGradient>
+
+<PCSX2PRLink authors="SternXD" prNums="12740,12944,13294">
+  ### Better Feature Parity Between Big Picture Mode and Qt Interface
+</PCSX2PRLink>
+
+SternXD has been on a mission to achieve feature parity between Big Picture Mode and the Qt interface. Thanks to his work, you can now adjust Network & HDD Settings, create a Memory Card, as well as login to RetroAchievements directly from Big Picture Mode!
+
+<Image src={require("./img/BPM_network.webp").default} />
+
+#### New icon for RetroAchievements Qt login dialog ([#13718](https://github.com/PCSX2/pcsx2/pull/13718))
+
+Alongside Big Picture Mode getting a new RetroAchievements login dialog, Qt's login dialog also gets a small facelift! It now shows the actual RetroAchievements icon instead of a generic login icon.
+
+<Image src={require("./img/RA_login.webp").default} />
+
+<PCSX2PRLink authors="TheTechnician27" prNums="13199">
+  ### Korean Game Titles
+</PCSX2PRLink>
+
+Despite having nearly 700 known NTSC-K serials, the GameDB has for a long time only had Latin versions of the titles. These titles were pulled from the NTSC-U/PAL name or from a Romaji version of the Japanese name. PCSX2 now has 100% complete Korean translation [on our Crowdin](https://crowdin.com/project/pcsx2-emulator) (at the time of writing), so this has been a long time coming. TheTechnician27 has been working on adding Korean game titles to most NTSC-K serials which did not previously have them.
+
+:::note
+If you have spotted any inaccuracies with the Korean game titles, please report such problem on [our GitHub issue](https://github.com/PCSX2/pcsx2/issues).
+:::
+
+<PCSX2PRLink authors="kamfretoz" prNums="12746">
+  ### Game Shortcut Creation Feature
+</PCSX2PRLink>
+
+KamFretoZ set his sights on adding the ability to create desktop (or start menu) shortcuts for your games from inside PCSX2. [Motivated by this GitHub issue](https://github.com/PCSX2/pcsx2/issues/8349) and lots of feedback on Discord, this was a very popular ask from users!
+
+:::info
+This feature is only available for Windows and Linux at this time.
+:::
+
+:::tip
+This Feature can be accessed by right-clicking the game you want to make the shortcut with on the game list and selecting "Create Game Shortcut".
```

Comment:
Lowercase "feature". Unsure if "right clicking" or "right-clicking", I'll leave that one to preference.